swagger/sales/coupons.json

```diff
@@ -0,0 +1,96 @@
+{
+  "host": "api.vchost.co",
+  "swagger": "2.0",
+  "info": {
+    "title": "API V3",
+    "version": "v3.0"
+  },
+  "paths": {
+    "/v3/sales/coupons": {
+      "get": {
+        "summary": "Get coupons api",
+        "description": "Get list of coupons",
+        "x-amazon-apigateway-integration": {
+          "responses": {
+            "default": {
+              "statusCode": "200"
+            }
+          },
+          "uri": "https://http://localhost:7100/v3/sales/coupons",
+          "httpMethod": "GET",
+          "type": "http_proxy"
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "translation missing: en.sales-api.coupons_message.tag"
+        ],
+        "security": [
+          {
+            "default": [
+              "business/payments"
+            ]
+          }
+        ],
+        "parameters": [
+          {
+            "name": "Authorization",
+            "in": "header",
+            "type": "string",
+            "required": true,
+            "description": "The format of the API authorization header should be: 'Authorization': \"Bearer API_TOKEN\"",
+            "x-example": "Bearer abc...123"
+          }
+        ],
+        "x-amazon-apigateway-auth": {
+          "type": "NONE"
+        },
+        "x-auth-type": "Application & Application User",
+        "responses": {
+          "200": {
+            "description": "Success",
+            "examples": {
+              "application/json": {
+                "success": true,
+                "data": {
+                  "success": true,
+                  "data": {
+                    "success": "true / false",
+                    "data": [
+                      {
```

Comment:
this is not the correct format for return. you should be returning the entity as a ref.

Reply:
Do you mean instead of data return  
`coupons: [{ ... coupons obj } , { ...coupons obj]`?

Reply:
yes.